api/users/views.py

```diff
@@ -18,11 +21,13 @@ class UserMixin(object):
 
     def get_user(self, check_permissions=True):
         obj = get_object_or_404(User, self.kwargs[self.node_lookup_url_kwarg])
-        if check_permissions:
+
+        if check_permissions & obj.is_disabled is False:
```

Comment:
More to the point - if it's not correct, then we need to write regression tests and fix it.

There shouldn't be parts of the codebase where we fear to tread. Step boldly, and if the boards give beneath your feet then it is time that the be reinforced.